packages/next/src/client/components/static-generation-async-storage.external.ts

```diff
@@ -16,12 +20,7 @@ export interface StaticGenerationStore {
   readonly isRevalidate?: boolean
   readonly isUnstableCacheCallback?: boolean
 
-  /**
-   * If defined, this function when called will throw an error postponing
-   * rendering during the React render phase. This should not be invoked outside
-   * of the React render phase as it'll throw an error.
-   */
-  readonly postpone: ((reason: string) => never) | undefined
+  prerenderState: null | PrerenderState
```

Comment:
It seems like the absence of `prerenderState` in the store indicates that the request is not associated with a static pre-render (ie, not static generation and PPR is not enabled). I'd suggest adding some comments here to describe this, as a `prerenderState === null` doesn't really tell me much besides "there is no prerender state".